### ISSUE TITLE
[Bug]: Live doesn't work

### ISSUE THREAD

**Issue by omarabdelaz1z** [CodeWithDennis/filament-select-tree#48]
### What happened?

When a field (a native select) value is set, the select-tree component does not function and is always disabled forever.

### How to reproduce the bug

```php
Forms\Components\Section::make('Store Info')->schema([
    Forms\Components\Select::make('store_code')
        ->options(Store::query()->pluck('name', 'code'))
        ->live()
        ->required()
        ->placeholder('Select Store'),
    SelectTree::make('categories')
        ->relationship('categories', 'name', 'parent_id')
        ->withCount()
        ->independent(false)
        ->enableBranchNode()
        ->searchable()->noSearchResultsMessage('No categories found')
        ->placeholder(
            fn(Get $get): string => !$get(
                'store_code'
            ) ? 'Select Store First..' : 'Select Categories..'
        )
        ->disabled(fn(Get $get): bool => empty($get('store_code'))),
])->columns(),                
```

### Package Version

3.1.12

### PHP Version

8.1

### Laravel Version

10.10

### Which operating systems does with happen with?

Linux

**Comment by CodeWithDennis** [comment]
@omarabdelaz1z I just released [v3.1.13](https://github.com/CodeWithDennis/filament-select-tree/releases/tag/v3.1.13) that should fix it.

**Comment by omarabdelaz1z** [comment]
Thanks alot!!
